components/Contributors.js

```diff
@@ -1,3 +1,5 @@
+import styles from './css/Contributors.module.css';
```

Comment:
What do you think about moving `Contributors.module.css` into the components folder? I prefer to have all the files next to each other. Not a big deal though.